modules/vue-i18n/vueSimpleRouter/plugins/i18nRoutes.ts

```diff
@@ -1,7 +1,8 @@
 import VUE_I18N_LOCALES from '#ioc/config/VUE_I18N_LOCALES'
 import VUE_I18N_ROUTE_PATHS from '#ioc/config/VUE_I18N_ROUTE_PATHS'
-
-export default (routes: any) => {
+let _isLayout = false
+export default (routes: any, isLayout = false) => {
```

Comment:
Boolean feature flags in function parameters are always a bad idea. Use options object

`export default (routes: any, { isLayout = false } = {}) => {`